client/state/query-cache.ts

```diff
@@ -0,0 +1,3 @@
+import { QueryCache } from 'react-query';
+
+export const queryCache = new QueryCache();
```

Comment:
Created our own query cache so we can look for cached results outside React hooks.